docs_src/security/Ch-SecretStore.md

```diff
@@ -17,1115 +29,259 @@ that you want to tightly control access to, such as API keys, passwords,
-will show the various Vault status (heartbeat and init/unseal states),
-coloring the boxes in green, orange or red depending on the level of
-importance (info, warning, error). By clicking each of the right side
-status indicators, more information will be accessible in order to
-better inspect any situation.
-
-![image](EdgeX-UI-Consul-01.png)
-
-As a practical example, we are going to navigate the Consul structure
-for Vault in order to check if the API Gateway (Kong) TLS certificate
-and private key were fetched and stored accordingly during the
-`vault-worker` process.
-
-First select `KEY/VALUE` menu, and then select `vault` root structure:
-
-![image](EdgeX-UI-Consul-02.png)
-
-We are now going to navigate deeper in the `vault` tree structure to
-reach and display the EdgeX Kong TLS assets. Continue by selecting
-`logical/`:
-
-![image](EdgeX-UI-Consul-03.png){.align-center width="348px"
-height="287px"}
-
-Then select `d7809b...` an arbitrary UID generated and created by Consul
-during Vault registration:
-
-![image](EdgeX-UI-Consul-04.png){.align-center width="377px"
-height="216px"}
-
-Select `edgex/`:
-
-![image](EdgeX-UI-Consul-05.png){.align-center width="419px"
-height="228px"}
-
-Select `pki/`:
-
-![image](EdgeX-UI-Consul-06.png){.align-center width="417px"
-height="222px"}
-
-Select `tls/`:
-
-![image](EdgeX-UI-Consul-07.png){.align-center width="418px"
-height="237px"}
+Additionally, Vault supports pluggable "secrets engines".
+EdgeX uses the Consul secrets engine to allow Vault to
+issue Consul access tokens to EdgeX microservices.
 
-Select `edgex-kong/`:
+In EdgeX, Vault's storage backend is the host file system.
+Vault has the capability to run in a high-availability mode
+by supporting replicated storage backends,
+but these high availabilty features are not currently enabled.
 
-![image](EdgeX-UI-Consul-08.png){.align-center width="423px"
-height="233px"}
-
-And we are now finally able to display the encrypted Vault secret
-containing the API Gateway (Kong) TLS server certificate and its
-corresponding private key. As you can see on the screenshot below the
-Vault key/value is encrypted and totally opaque to Consul, the Vault
-data encryption key (DEK) would be necessary to decrypt these secrets.
-Each Vault secret is encrypted before being transmitted to Consul
-node(s).
-
-![image](EdgeX-UI-Consul-09.png)
-
-### Shell Access to Consul Container and Using Consul CLI
-
-``` bash
-sh> docker exec -it -e PS1='\u@\h:\w \$ ' edgex-core-consul sh
-
-root@edgex-core-consul:/ # consul members
-Node               Address         Status  Type    Build  Protocol  DC   Segment
-edgex-core-consul  127.0.0.1:8301  alive   server  1.1.0  2         dc1  <all>
+## Start the Secret Store
 
-root@edgex-core-consul:/ # consul catalog nodes
-Node               ID        Address    DC
-edgex-core-consul  e49af36a  127.0.0.1  dc1
+The EdgeX secret store is started by default when using the Docker Compose
+scripts found at
```

Comment:
```suggestion
files found at
```

Reply:
Is there another document that I can refer people to that tells them how to start the EdgeX framework.  Seems silly to keep duplicating this.